packages/data-store/src/data-store.ts

```diff
@@ -142,4 +170,81 @@ export class DataStore implements IAgentPlugin {
 
     return presentationEntity.raw
   }
+
+  async dataStoreSaveMediationPolicy({ did, policy }: IDataStoreSaveMediationPolicyArgs): Promise<string> {
+    const db = await getConnectedDb(this.dbConnection)
+    const existingDid = await db.getRepository(MediationPolicy).findOne({ where: { did } })
+    if (existingDid) throw new Error('already_exists: Mediation policy already exists for provided did')
+    const saveResult = await db.getRepository(MediationPolicy).save({ did, policy })
+    return saveResult.did
+  }
+
+  async dataStoreRemoveMediationPolicy({
+    did,
+  }: IDataStoreRemoveMediationPolicyArgs): Promise<RemoveMediationPolicyResult> {
+    const db = await getConnectedDb(this.dbConnection)
+    const existingEntry = await db.getRepository(MediationPolicy).findOne({ where: { did } })
+    if (!existingEntry) return null
+    await db.getRepository(MediationPolicy).remove(existingEntry)
+    return existingEntry.did
+  }
+
+  async dataStoreGetMediationPolicies({
```

Comment:
```suggestion
/**
 * This is used internally when global policy `isMediateDefaultGrantAll` is:
 *  - true: with policy = MediatePolicy.DENY [All DIDs allowed except those explicitly denied]
 *  - false: with policy = MediatePolicy.ALLOW [All DIDs denied except those explicitly allowed]
 *
 * @beta This API may change in future versions without a BREAKING CHANGE notice.
 */
  async dataStoreGetMediationPolicies({
```

Reply:
Thanks for this. Good suggestion, I have included it in the PR.